get-started/test-drive.md

```diff
@@ -0,0 +1,174 @@
+    <li class="tab-link" data-tab="tab-install-vscode">VS Code</li>
+    <li class="tab-link" data-tab="tab-install-terminal">Terminal + editor</li>
+</ul>
+
+<div id="tab-install-androidsstudio" class="tabs__content current" markdown="1">
+
+*Android Studio:* A complete, integrated IDE experience for Flutter. 
+
+## Create new app {#create-app}
+
+   1. Select **File>New Flutter Project**
+   1. Under **Flutter SDK path**, select **Install SDK...**, and enter the
+      location where you want to install the Flutter SDK
+   1. Click **OK**
+   1. Wait for Android Studio to install the SDK, and create the project.
+
+The above command creates a Flutter project directory called `myapp` that contains a simple demo
+app that uses [Material Components](https://material.io/guidelines/).
+
+In the project directory, the code for your app is in `myapp/lib/main.dart`.
+
+## Run the app
+
+   1. Locate the main Android Studio toolbar:<br>
+      ![Main IntelliJ toolbar](/images/intellij/main-toolbar.png)
+   1. In the **target selector**, select an Android device for running the app.
+      If none are listed as available, select **Tools>Android>AVD Manager** and
+      create one there. For details, see [Managing
+      AVDs](https://developer.android.com/studio/run/managing-avds.html).
+   1. Click the **Run icon** in the toolbar, or invoke the menu item **Run >
+      Run**.
+   1. If everything works, you should see your starter app on your device or
+      simulator:<br>
+      ![Starter App on Android](/images/flutter-starter-app-android.png)
+
+## Try a hot reload
+
+Flutter offers a fast development cycle with _hot reload_, the ability to reload
+the code of a live running app without restarting or losing app state. Simply
+make a change to your source code, tell your IDE or command-line tool that you
+want to hot reload, and see the change in your simulator, emulator, or device.
+
+  1. Change the string<br>`'You have pushed the button this many times:'`
+     to<br>`'You have clicked the button this many times:'`
+  1. Do not press the 'Stop' button; let your app continue to run.
+  1. To see your changes invoke **Save All** (`cmd-s` / `ctrl-s`), or click the
+     **Hot Reload button** (the button with the lightning bolt icon).
+
+You should see the updated string in the running app almost immediately.
+
+</div>
+
+<div id="tab-install-vscode" class="tabs__content" markdown="1">
+
+*VS Code:* A light-weight editor with Flutter run and debug support.
+
+## Create new app
+
+  1. Start VS Code
+  1. Invoke **View>Command Palette...**
+  1. Type 'flutter', and select the **'Flutter: New Project'** action
+  1. Enter a project name (e.g. `firstapp`), and press Enter
+  1. Specify a location to place the project, and press the blue OK button
+  1. Wait for the project creation to continue, and the `main.dart` file to appear 
+
+The above command creates a Flutter project directory called `myapp` that contains a simple demo
```

Comment:
This says `myapp` but a few lines up it says `firstapp`; I think they're supposed to be the same?

Reply:
Good catch, fixed